starlark/testdata/proto.star

```diff
@@ -0,0 +1,14 @@
+# Tests of the experimental 'lib/proto' module.
+
+load("assert.star", "assert")
+load("proto.star", "proto")
+
+schema = proto.file("google/protobuf/descriptor.proto")
```

Comment:
Thanks. I think this is a reasonable starting point. At some point we will likely want to define our own .proto descriptors to exercise a larger range of features (and to be a little less "meta"). In that case, add the .proto file to the testdata tree, and define a go-generate command that runs protoc to update the .pb.go file from it, and commit the .pb.go as a subpackage of the test. 

Make sure the //go:generate directive is accompanied by instructions for anyone that doesn't have protoc installed on their machine, something like this:

```
/*
One-time installation:
curl --location --silent -o protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v3.19.4/protoc-3.19.4-linux-x86_64.zip 
unzip protoc.zip
go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28.1 
*/
//go:generate protoc [-flags] test.proto
```
The google3 version of this package already contains a reasonable test suite, so I suggest you export that.